nin/dasBoot/RootNode.js

```diff
@@ -0,0 +1,34 @@
+(function(NIN) {
+  'use strict';
+
+  class RootNode extends NIN.Node {
+    constructor(id, options) {
+      super(id, {
+        inputs: {screen: new NIN.TextureInput()}
+      });
+
+      this.camera = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1); 
+      this.scene = new THREE.Scene();
+      this.quad = new THREE.Mesh(new THREE.PlaneBufferGeometry(2, 2), null);
+      this.scene.add(this.quad);
+      this.material = new THREE.MeshBasicMaterial();
+    }
+
+    render(renderer) {
+      if(renderer.overrideToScreenTexture) {
+        this.quad.material = new THREE.MeshBasicMaterial({
+          map: renderer.overrideToScreenTexture
```

Comment:
Yes and no. Just rendering a plain ol' THREE.Scene directly to the screen does not require rendering a quad. Chaining renderings with the EffectComposer in the old layer paradigm would render all but the last pass into a rendertarget so that it could be fed as a texture into the next pass. For shaderpasses and copypasses this quad technique was used. In effect, this last pass of the rendering chain in the new system is a copypass, since it renders an incoming texture right to the screen.